app/api/web.go

```diff
@@ -57,11 +57,10 @@ func (s *Server) getFeedPageCtrl(w http.ResponseWriter, r *http.Request) {
 
 	w.Header().Set("Content-Type", "text/html; charset=utf-8")
 	w.WriteHeader(http.StatusOK)
-	_, _ = w.Write(data.([]byte)) //nolint
```

Comment:
> Use //nolint instead of // nolint because machine-readable comments should have no space by Go convention. – https://github.com/golangci/golangci-lint#nolint

Reply:
@sgaynetdinov thanks for finding this! Would you be so kind as to adjust existing comments in your next PR?